baybe/searchspace/core.py

```diff
@@ -381,6 +379,13 @@ def transform(
         return comp_rep
 
 
+def to_searchspace(
+    x: Parameter | SubspaceDiscrete | SubspaceContinuous | SearchSpace, /
+) -> SearchSpace:
+    """Convert a parameter/subspace into a search space (with search space passthrough)."""  # noqa: E501
```

Comment:
This method should have a proper docstring which will also get rid of the noqa.

Reply:
Ok, if we are agreeing that one liners for such "basically just passthrough methods" is fine, then I am happy with that.